packages/gatsby-plugin-emotion/src/gatsby-ssr.js

```diff
@@ -1,23 +1,25 @@
-import React from "react"
-import { renderToString } from "react-dom/server"
-import { extractCritical } from "emotion-server"
+const React = require(`react`)
+const { renderToString } = require(`react-dom/server`)
+const { extractCritical } = require(`emotion-server`)
+
+const { wrapElement } = require(`./wrap-element`)
 
 exports.replaceRenderer = ({
   bodyComponent,
   replaceBodyHTMLString,
   setHeadComponents,
 }) => {
-  const { html, ids, css } = extractCritical(renderToString(bodyComponent))
-
-  const criticalStyle = <style dangerouslySetInnerHTML={{ __html: css }} />
-  const criticalIds = (
```

Comment:
Same thing here.

Reply:
v10 uses the `data-emotion-css` attribute for this now, rather than a string on the `window` object, as far as I understand it. Might need Mitchell's blessing here too, though.

Reply:
This is correct.